packages/core/src/js/client.ts

```diff
@@ -86,14 +86,8 @@ export class ReactNativeClient extends BaseClient<ReactNativeClientOptions> {
   /**
    * Sends user feedback to Sentry.
    */
-  public captureUserFeedback(feedback: UserFeedback): void {
-    const envelope = createUserFeedbackEnvelope(feedback, {
-      metadata: this._options._metadata,
-      dsn: this.getDsn(),
-      tunnel: undefined,
-    });
-    // eslint-disable-next-line @typescript-eslint/no-floating-promises
-    this.sendEnvelope(envelope);
+  public captureFeedback(feedback: SendFeedbackParams): void {
+    captureFeedbackApi(feedback);
```

Comment:
[`captureFeedbackApi` dynamically gets client](https://github.com/getsentry/sentry-javascript/blob/dafd51054d8b2ab2030fa0b16ad0fd70493b6e08/packages/core/src/feedback.ts#L8), we should not call in in a client as it could cause unexpected result.

Client methods has to use this to captureEvent.

Maybe we could just deprecate this old API?

Reply:
Makes sense. I will remove the client implementation and move this on the sdk as per [your other comment](https://github.com/getsentry/sentry-react-native/pull/4320#discussion_r1862033816).